test/pseudolocalization_test.rb

```diff
@@ -53,4 +53,11 @@ def test_it_works_with_liquid_tags
   def test_it_works_with_templates
     assert_equal('Ḥḛḛḽḽṓṓ, ẁṓṓṛḽḍ %{firstname} %{lastname}!', @backend.translate(:en, 'Hello, world %{firstname} %{lastname}!', {}))
   end
+
+  def test_it_allows_ignoring_cetain_keys
+    @backend.ignores = ['Ignore*', /Clifford.$/]
+
+    assert_equal('Ignore me, World!', @backend.translate(:en, 'Ignore me, World!', {}))
+    assert_equal('Ignore me, as well Clifford!', @backend.translate(:en, 'Ignore me, as well Clifford!', {}))
```

Comment:
Can we add one non-ignore assertion here to ensure the `ignores` feature works 100% as intended?
```ruby
# maybe this one from above?
assert_equal(['Ḥḛḛḽḽṓṓ, ẁṓṓṛḽḍ!'], @backend.translate(:en, ['Hello, world!'], {}))
```

Reply:
Addressed